astropy/cosmology/parameters.py

```diff
@@ -150,4 +144,5 @@
                "Table 1 (WMAP + BAO + SN ML).")
 )
 
-available = tuple(k for k in locals() if not k.startswith('_'))
+# If new parameters are added, this list must be updated
```

Comment:
Also, in either case, since `locals()` returns a dictionary, it will need to look like the following:
```
all_locals = locals()
available = [k for k in all_locals if isinstance(all_locals[k], WhateverClass)]
```